internal/venice-common/src/main/java/com/linkedin/venice/ConfigKeys.java

```diff
@@ -1190,6 +1190,14 @@ private ConfigKeys() {
    */
   public static final String ENABLE_INCREMENTAL_PUSH_FOR_HYBRID_ACTIVE_ACTIVE_USER_STORES =
       "enable.incremental.push.for.hybrid.active.active.user.stores";
+
+  /**
+   * We will use this config to determine whether we should enable separate real-time topic for incremental push enabled stores.
+   * If this config is set to true, we will enable separate real-time topic for incremental push enabled stores.
+   */
+  public static final String ENABLE_SEPARATE_REAL_TIME_TOPIC_FOR_STORE_WITH_INCREMENTAL_PUSH =
+      "enable.separate.real.time.topic.for.store.with.incremental.push";
```

Comment:
Could you also add an admin tool command to this config?

Reply:
I'm thinking whether it makes more sense to have the consistent naming. We have used the "separate topic enabled" in other places. How about renaming this one too?

Reply:
This is cluster-wide config to turn on this feature when we are confident for enabling this feature for new stores with incremental push enabled. So for store level config, there is only one with `separte_realtime_topic_enabled` flag to be turned on.